NastassiaBon/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+def data_extraction
+  file = File.new('data.txt')
+  file.read
+end
+
+def mean(town, data)
+  return -1.0 unless data.include? town
+
+  serch_town(town, data).sum / 12
+end
+
+def variance(town, data)
+  return -1.0 unless data.include? town
```

Comment:
Brackets

Reply:
Why float -1?

Reply:
According to the task on the codewars, we are given the opportunity to choose -1 or -1.0, but both options will be correct and will not greatly affect the task